cime_config/namelist_definition_blom.xml

```diff
@@ -3842,6 +3842,7 @@
       <value blom_n_deposition="yes" blom_ndep_scenario="1850" ocn_grid="tnx0.25v4">$DIN_LOC_ROOT/ocn/blom/bndcon/ndep_1850_CMIP6_tnx0.25v4_20190912.nc</value>
       <value blom_n_deposition="yes" blom_ndep_scenario="2000" ocn_grid="tnx0.25v4">$DIN_LOC_ROOT/ocn/blom/bndcon/ndep_2000_CMIP6_tnx0.25v4_20200826.nc</value>
       <value blom_n_deposition="yes" blom_ndep_scenario="hist" ocn_grid="tnx0.25v4">$DIN_LOC_ROOT/ocn/blom/bndcon/ndep_185001-201412_tnx0.25v4_20190705.nc</value>
+      <value blom_n_deposition="yes" blom_ndep_scenario="ssp"  ocn_grid="tnx0.25v4">$DIN_LOC_ROOT/ocn/blom/bndcon/ndep_201501-210012-${BLOM_NDEP_SCENARIO}_tnx0.25v4_20250325.nc</value>
```

Comment:
Hi @YanchunHe , thanks for putting your changes in! I here personally would prefer to remove the links on betzy and just write here `.../bnd/ndeposition_201501-210012-${BLOM_NDEP_SCENARIO}_tnx0.25v4_20250325.nc`. But maybe @TomasTorsvik and @JorgSchwinger are differently opinionated (for explanation: with the separate `nhx` and `noy` inclusion into those files, I started to name them `ndeposition` instead of purely `ndep`, which gave me some indication of the differences...) Maybe you can comment?

Reply:
I would also suggest to have the naming here consistently `ndeposition_...`. But maybe the links on betzy are needed for backwards compatibility?

Reply:
I was actually using an older version of BLOM 1.6.9 (NorESM2_3_develop), so it looking for the name `ndep_xxx-xxx*.nc`. So this is mainly for backward compatibility (if we think it is necessary). Or we can use `ndeposition_` here for the latest BLOM version, but commit changes to the but use `ndep_*.nc` for `NorESM2_3_develop`?